src/reformat_libyuv.c

```diff
@@ -341,6 +341,27 @@ avifResult avifImageYUVToRGBLibYUV(const avifImage * image, avifRGBImage * rgb)
     return AVIF_RESULT_NOT_IMPLEMENTED;
 }
 
+// These defines are used to create a NULL reference to libyuv functions that
+// did not exist prior to a particular version of libyuv.
+#if LIBYUV_VERSION < 1838
+#define I422ToRGB565Matrix NULL
+#endif
+#if LIBYUV_VERSION < 1813
+#define I422ToARGBMatrixFilter NULL
+#define I420ToARGBMatrixFilter NULL
+#define I210ToARGBMatrixFilter NULL
+#define I010ToARGBMatrixFilter NULL
+#endif
+#if LIBYUV_VERSION < 1780
+#define I410ToARGBMatrix NULL
+#endif
+#if LIBYUV_VERSION < 1756
+#define I400ToARGBMatrix NULL
+#endif
+
+#define RGB_PIXEL_FORMATS 7
+#define YUV_PIXEL_FORMATS 5
```

Comment:
Optional: The lookup tables depend on the current `avifRGBFormat` and `avifPixelFormat` enum values, not just how many enum values there are.

Since these two enum types are not based on some standard, the enum values may change until we freeze the API. It would be good to figure out a way to assert the asseumptions,